examples/js/math/Lut.js

```diff
@@ -194,7 +196,31 @@ THREE.Lut.prototype = {
 
 		}
 
-		this.legend.ctx.putImageData( imageData, 0, 0 );
+		ctx.putImageData( imageData, 0, 0 );
+
+		return canvas;
+
+	},
+
+	setLegendOn: function ( parameters ) {
+
+		parameters = parameters || defaultLegendParameters;
+
+		this.legend = {};
+
+		this.legend.layout = parameters.layout || defaultLegendParameters.layout;
+
+		this.legend.position = parameters.position || defaultLegendParameters.position;
+
+		this.legend.dimensions = parameters.dimensions || defaultLegendParameters.dimensions;
+
+		var canvas = this.createCanvas();
+
+		document.body.appendChild( canvas );
+
+		this.legend.canvas = canvas;
+
+		this.legend.texture = new THREE.Texture( canvas );
 		this.legend.texture.needsUpdate = true;
 
 		this.legend.legendGeometry = new THREE.PlaneBufferGeometry( this.legend.dimensions.width, this.legend.dimensions.height );
```

Comment:
I don't it's necessary to create a geometry and a material only because the legend is activated. This should be done once and only updated.

Reply:
IMO, the legend code can be removed.

Reply:
Legend code has been removed.